File: eco-web/src/app/dto/cartDto.ts
import { Product } from "../domain/product";

export class CartDto {

    constructor(private username:string, private product:Product){
        this.username = username;
        this.product = product;
    }

    getProduct(){
        return this.product;
    }
    getUsername(){
        return this.username;
    }
}

File: eco-back/src/main/java/ma/fgs/product/domain/dto/CartDto.java
package ma.fgs.product.domain.dto;

import ma.fgs.product.domain.Product;

public class CartDto {

	private String username;
	
	private Product product;

	public CartDto() {
		
	}
	
	public CartDto(String username, Product product) {
		super();
		this.username = username;
		this.product = product;
	}

	public String getUsername() {
		return username;
	}

	public void setUsername(String username) {
		this.username = username;
	}

	public Product getProduct() {
		return product;
	}

	public void setProduct(Product product) {
		this.product = product;
	}
	
}
